### DISCUSSION TITLE
Some new requirement should be added

### DISCUSSION THREAD

**Discussion by abubakarphp** [voidbydefault/EasyApplyBot#34] [General]
- How Users can apply for jobs other than easy apply.
- Link can be saved as a proper hyperlink in the output and failed sheet.
- How complete job description can be added into the output and failed csv sheet.
- How some extra questions can be added into the config and easyapply file for autofil.

**Comment by voidbydefault** [comment]
> How Users can apply for jobs other than easy apply.

How do you propose to automate? Non-Easy Apply jobs take you to third party websites and every website has its unique way of submitting applications. It's just too many to automate one by one. You can find some paid Chrome extensions that claim to work. Never tried, can't recommend any.

> Link can be saved as a proper hyperlink in the output and failed sheet.

Bot already does so. Check your bot's folder for `*.csv` files.

> How complete job description can be added into the output and failed csv sheet.

Bot currently records employer name, URL, title, and time it applied in `*.csv` files in its folder.

> How some extra questions can be added into the config and easyapply file for autofil.

Many are added but let me know if you think there is a frequently asked question that's missing. You can check `unprepared questions.csv` in bot's directory.
